### DISCUSSION TITLE
Does listening in Spotube increase Youtube view of the video?

### DISCUSSION THREAD

**Discussion by vanbang9710** [KRTirtho/spotube#822] [Q&A] [answered]
Afaik, Spotube uses Youtube api, hence the title question

**Comment by ksharizard** [comment, marked as answer]
Nope. Using the Youtube API in spotube does not increase the view of the video.

**Comment by ildar** [comment]
So that's also true for LibreTube/NewPipe/SkyTube etc.?

**Reply by ksharizard** [reply]
Yes, I personally tested it with the Libretube and Newpipe client and the view count of one of my videos did not increase.
